src/site/constants/buckets.js

```diff
@@ -1,20 +1,6 @@
-const {
-  DEVELOPMENT,
-  PRODUCTION,
-  STAGING,
```

Comment:
Are these useful in any other context? Seems confusing to mix these and the `VAGOV*` variants.

Reply:
Confusing is right. There was a [recent devops PR](https://github.com/department-of-veterans-affairs/devops/pull/7296) that prompted me to do this after my own confusion trying to understand what was going on with the s3 buckets.

To my knowledge they aren't useful in any other context.  I'll let @rianfowler take a look before I merge though.

Reply:
They were the vets.gov environments - dev|staging|www.vets.gov - and for a period they coexisted with the va.gov environments.